sendsmaily_subscribe/sendsmaily_subscribe.module

```diff
@@ -97,37 +96,21 @@ function sendsmaily_subscribe_configuration_page() {
     '#required' => TRUE,
   );
 
-  return system_settings_form($form);
-}
+  $form['sendsmaily_success_url'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Success URL'),
+    '#default_value' => variable_get('sendsmaily_success_url', ''),
+    '#description' => t('For example: http://example.com Leave blank for redirect back'),
```

Comment:
I think we can improve the description. What about "For example https://example.com/welcome. Leave blank for redirect back to page where form was submitted from."?